From 61da97656d28ea63a22ded1f9b42b3c2447cb8cf Mon Sep 17 00:00:00 2001
From: Weifeng Liu <weifeng.liu@intel.com>
Date: Mon, 22 Jan 2024 09:59:32 +0000
Subject: [PATCH 14/14] virtio_ivshmem: Use empty PCI id table

Allows front-end to boot up without back-end device model running.

Signed-off-by: Weifeng Liu <weifeng.liu@intel.com>
---
 drivers/virtio/virtio_ivshmem.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/virtio/virtio_ivshmem.c b/drivers/virtio/virtio_ivshmem.c
index f812f3214618..c8e77d0326a7 100644
--- a/drivers/virtio/virtio_ivshmem.c
+++ b/drivers/virtio/virtio_ivshmem.c
@@ -1014,7 +1014,7 @@ MODULE_DEVICE_TABLE(pci, virtio_ivshmem_id_table);
 
 static struct pci_driver virtio_ivshmem_driver = {
 	.name		= DRV_NAME,
-	.id_table	= virtio_ivshmem_id_table,
+	.id_table	= NULL,
 	.probe		= virtio_ivshmem_probe,
 	.remove		= virtio_ivshmem_remove,
 };
-- 
2.17.1

